bun-tests/fake-snippets-api/routes/orders/order-with-circuit-json-or-package-release.test.ts

```diff
@@ -0,0 +1,86 @@
+import { getTestServer } from "bun-tests/fake-snippets-api/fixtures/get-test-server"
+import { test, expect } from "bun:test"
+
+const setupTestServer = async (
+  withCircuitJson: boolean,
+  withPackageRelease: boolean,
+) => {
+  return await getTestServer({
+    withCircuitJson,
+    withPackageRelease,
+  })
```

Comment:
Dont wrap functions like this! It increases the goto depth and reduces readability!